pkg/client/clientset/versioned/fake/register.go

```diff
@@ -40,14 +40,14 @@ var localSchemeBuilder = runtime.SchemeBuilder{
 // AddToScheme adds all types of this clientset into the given scheme. This allows composition
 // of clientsets, like in:
 //
-//   import (
-//     "k8s.io/client-go/kubernetes"
-//     clientsetscheme "k8s.io/client-go/kubernetes/scheme"
-//     aggregatorclientsetscheme "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/scheme"
-//   )
+//	import (
+//	  "k8s.io/client-go/kubernetes"
```

Comment:
It feels like these files shouldn't be modified, so it causes the automatic check to fail

Reply:
these are happening automatically when doing the make command don't remember the exact command. I will exclude these changes manully. Hopefully, that would work.

Reply:
But currently the CI check always failed, and I suspect this may be the cause. There is no way for us to merge in if the check fails. Or you can change the go version to 1.18